BuildForSDG/SmeVest-Frontend
# chore: configure heroku staging and modify readme

## Description
- Added the required scripts and package for heroku staging environemt deployment
-  Removed the old codacy badge from readme
- modified app.properties file
- Added configuration for choosing the right API_ENDPOINT for various node environments.

Finishes #27

## How Has This Been Tested?
- Update your `.env` file with `.env.example`  details
- Run `yarn install`
- Use `yarn start:dev` to run on your local machine
- To check online Using this link:
https://smevest-frontend-staging.herokuapp.com/

## Checklist:
